README.md

```diff
@@ -28,6 +28,8 @@ setting up the rest of the OceanWATERS repositories:
 ## Getting Started
 - [Install software prerequisites](oceanwaters/doc/setup_dev_env.md)
 - [Download and build OceanWATERS](oceanwaters/doc/setup_oceanwaters.md)
+  - Default build instruction use ROS Melodic, but you will also find an option
+    to use experimental ROS Noetic support.
```

Comment:
The requirements page state that you need to use Ubuntu 18.04 and melodic. The note needs to proceed the requirements page in my opinion. I just moved your note one level up but consider rephrasing it to fit the new location.
```suggestion
## Getting Started
Default build instruction use ROS Melodic, but you will also find an option to use experimental ROS Noetic support.
- [Install software prerequisites](oceanwaters/doc/setup_dev_env.md)
- [Download and build OceanWATERS](oceanwaters/doc/setup_oceanwaters.md)
```